### DISCUSSION TITLE
What is the most stable version at the moment for Android and iOS?

### DISCUSSION THREAD

**Discussion by httpwebmedia** [airsdk/Adobe-Runtime-Support#702] [General]
It would be great if the community and Harman share their experience.

For instance, we have so many reports on android for the error **"Input dispatching timed out (Waiting to send non-key event because the touched window has not finished processing certain input events that were delivered to it over 500.0ms ago. Wait queue length: 1. Wait queue head age: 5903.8ms.), VisibleToUser"**

I think it would be very interesting for us to know the most stable version for each platform.

**Comment by ajwfrost** [comment]
Excellent idea..

The issue that I sometimes see is that when we fix something for some applications, it breaks things for others. This is particularly true of audio on Android which I believe is behind a lot of the ANRs, it was refactored not that long ago by Adobe and we're still seeing some issues around it. So the stability is based upon what you're actually doing and how you're doing it ... I would say, if people are having a low ANR rate and yet are still using audio, it would be great if they could clarify how they're using audio (e.g. playing short snippets from embedded data, or from local files, etc..) so that we can try to build up a pattern of what works vs what may be causing issues.

BTW this should perhaps be under 'discussions' rather than 'issues' ...

thanks
